### ISSUE TITLE
[Bug] (Desktop) Operation time-out on shenron steps 7, 8 & 9 

### ISSUE THREAD

**Issue by Oran-Dan** [impierce/identity-wallet#215]
### Description

On my desktop I get the following error message after step 7, 8 or 9.
[2024-04-29][15:26:46][ERROR][identity_wallet::command] Operation timed out
[2024-04-29][15:26:46][ERROR][log@http://localhost:5173/node_modules/.vite/deps/@tauri-apps_plugin-log.js:59:20] timed out

On my phone iOS it does work.

### Hardware Specification

Operating system: Ubuntu 22.04
RAM: 
Cores:
Device: 

### Steps to Reproduce the Bug

1. Start the app and click step 7, 8 or 9
2. You will be returned to the homescreen where nothing changed
3. Check the console logs for the error message

### Expected Behaviour

The steps should be completed

### Actual Behaviour

The operation fails and you are returned to the homepage.

### Errors

[2024-04-29][15:26:46][ERROR][identity_wallet::command] Operation timed out
[2024-04-29][15:26:46][ERROR][log@http://localhost:5173/node_modules/.vite/deps/@tauri-apps_plugin-log.js:59:20] timed out

**Comment by JelleMillenaar** [comment]
Superseded by https://github.com/impierce/identity-wallet/issues/299 as that will fix the issue
